package/test_requirements.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements.txt
-kedro[pandas.ParquetDataSet, pandas.CSVDataSet, plotly.JSONDataSet]>=0.17.0  
+kedro >=0.17.0  
+kedro-datasets~=1.0
```

Comment:
I guess you still only need these datasets `[pandas.ParquetDataSet, pandas.CSVDataSet, plotly.JSONDataSet]` right? So you can still specify that in the import.